internal/exec/resolvable/packer.go

```diff
@@ -130,8 +130,11 @@ func (b *execBuilder) makeStructPacker(values common.InputValueList, typ reflect
 	var fields []*structPackerField
 	for _, v := range values {
 		fe := &structPackerField{field: v}
+		fx := func(n string) bool {
+			return strings.EqualFold(n, v.Name.Name) || strings.EqualFold(camelize(n), camelize(v.Name.Name))
```

Comment:
`strings.EqualFold(n, v.Name.Name)` shouldn't be needed either, should it?

As far as I can tell, for all inputs where `strings.EqualFold(camelize(n), camelize(v.Name.Name))` is false, ``strings.EqualFold(n, v.Name.Name)` would be false too.